app/views/programming_groups/index.html.slim

```diff
@@ -0,0 +1,34 @@
+h1 = ProgrammingGroup.model_name.human(count: 2)
+
+- if params[:exercise_id].nil?
+  = render(layout: 'shared/form_filters') do |f|
+    .col-auto
+      = f.label(:exercise_id_eq, t('activerecord.attributes.programming_group.exercise'), class: 'visually-hidden form-label')
+      = f.collection_select(:exercise_id_eq, Exercise.with_programming_groups, :id, :title, class: 'form-control', prompt: t('activerecord.attributes.programming_group.exercise'))
+
+.table-responsive
+  table.table.mt-4 class="#{@programming_groups.present? ? 'sortable' : ''}"
+    thead
+      tr
+        th = sort_link(@search, :id, t('activerecord.attributes.programming_group.programming_group_name'))
+        th = sort_link(@search, :exercise_id, t('activerecord.attributes.programming_group.exercise'))
+        th = sort_link(@search, :user_id, t('activerecord.attributes.programming_group.member'))
+        th = sort_link(@search, :count, t('activerecord.attributes.programming_group.member_count'))
```

Comment:
Same as above: Sorting in the backend is not that trivial, even though possible. Hence, I would probably skip it for now, this might be some feature in the future. If desired, we could add a filter (e.g., a dropdown to see those with exactly 2 members and those with more), but since we currently limited all groups to two, I wouldn't add this filter either).